ml_core/algorithms/id_embedding.py

```diff
@@ -0,0 +1,42 @@
+import numpy
+
+# todo(guillemdb): maybe refactor in mining
```

Comment:
I think that everything that does not depend on `gitbase` project should be in this repo. So let's keep it.
```suggestion
```